### PR TITLE
[MVP] Rackspace Migration Manager: Add article to map Rackspace to AWS resources

### LINKED ISSUE #2090: [MVP] Rackspace Migration Manager: Create article to map Rackspace resources
to AWS resources.

Reverse engineer this existing article to show Rackspace to AWS (as opposed to AWS to Rackspace):https://support.rackspace.com/how-to/mapping-of-amazon-web-services-resources-to-rackspace-resources/

### PR DESCRIPTION
**Pull request by cyrichardson**
Fixes #2090

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-c20a6fc09e/how-to/)

**Comment by cyrichardson** [comment]
@amitgandhinz Please review when you have time and provide your comments in a normal github review of the file. I've also asked Trent Gillaspie to review but am not able to @ him here for some reason.

**Comment by amitgandhinz** [comment]
@trentgillaspie

File: content/rackspace-migration-manager/map-rackspace-to-AWS.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+| Rackspace product | Corresponding AWS product |
+| --- | --- |
+| [Cloud Servers product](http://www.rackspace.com/cloud/servers)<br />[Cloud Servers API](https://developer.rackspace.com/docs/cloud-servers/v2/developer-guide/) | Amazon EC2 |
+| [Cloud Load Balancers product](http://www.rackspace.com/cloud/load-balancing)<br />[Cloud Load Balancers API](https://developer.rackspace.com/docs/cloud-load-balancers/v1/developer-guide/) | Elastic Load Balancing|
+| [Rackspace Cloud Big Data Platform product](https://www.rackspace.com/cloud/big-data) | Amazon EMR |
+| [Cloud Networks product](https://www.rackspace.com/cloud/networks) | Amazon VPC |
+| [Cloud DNS product](https://www.rackspace.com/cloud/dns) | Amazon Route 53 |
+| [RackConnect](https://www.rackspace.com/cloud/hybrid/rackconnect) | AWS Direct Connect |
+| [Cloud Files product](http://www.rackspace.com/cloud/files)<br />[Cloud Files API](https://developer.rackspace.com/docs/cloud-files/v1/developer-guide/) | Amazon S3 |
+| [Cloud Backup product](https://www.rackspace.com/cloud/backup) | Amazon Glacier |
```
Comment (amitgandhinz) [inline content/rackspace-migration-manager/map-rackspace-to-AWS.md]: Amazon doesn't have an equivalent product to Cloud Backup.  Glacier is just long term storage at a cheaper rate than S3, but not really the same objective as a scheduled file level backup tool.

File: content/rackspace-migration-manager/map-rackspace-to-AWS.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+| [Cloud Block Storage product](http://www.rackspace.com/cloud/block-storage)<br />[Cloud Block Storage API](https://developer.rackspace.com/docs/cloud-block-storage/v1/developer-guide/) | Amazon EBS |
+| Hard Drive Import/Export | AWS Import/Export |
+| Multiple Partner solutions available | AWS Storage Gateway |
+| [Rackspace CDN product](http://www.rackspace.com/cloud/cdn-content-delivery-network)<br />[Rackspace CDN API](https://developer.rackspace.com/docs/cdn/v1/developer-guide/) | Amazon CloudFront |
+| [Cloud Databases product](https://www.rackspace.com/cloud/databases)<br />[Cloud Databases API](https://developer.rackspace.com/docs/cloud-databases/v1/developer-guide/) | Amazon RDS |
+| Object Rocket (MongoDB - open) | Amazon DynamoDB |
+| Partner solutions - [Mailgun](https://www.mailgun.com/) |
```
Comment (amitgandhinz) [inline content/rackspace-migration-manager/map-rackspace-to-AWS.md]: Amazon SNS could be the closest equivalent here.

**Comment by trentgillaspie** [comment]
@amitgandhinz, great point re: all products versus just those in scope for each phase of the migration tooling rollout. We don't want to mislead customers to think everything is migrateable in our first release, and this documentation will fall directly in line with the Migration Manager. @cyrichardson, let's just stick to Cloud Servers, Cloud Networks, and Cloud Block Storage (as attached volumes are accounted for in the initial release). Thanks!

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c3c002a575/how-to/)

File: content/rackspace-migration-manager/map-rackspace-to-AWS.md (hunk 1 above)
Comment (cyrichardson) [inline content/rackspace-migration-manager/map-rackspace-to-AWS.md]: Removed Cloud Backup

File: content/rackspace-migration-manager/map-rackspace-to-AWS.md (hunk 2 above)
Comment (cyrichardson) [inline content/rackspace-migration-manager/map-rackspace-to-AWS.md]: Remove Mailgun

**Comment by cyrichardson** [comment]
Review comments applied. @amitgandhinz and @trentgillaspie - Please recheck.
Remove all products except Cloud Servers, Cloud Networks, and Cloud Block Storage.

**Comment by trentgillaspie** [comment]
Looks good to me. Thanks, @cyrichardson!

**Comment by cyrichardson** [comment]
Moving to blocked status until product launch.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9baf9ddcec/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-830b722383/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-068c347b5c/how-to/).

**Comment by cyrichardson** [comment]
Ready for MVP launch

**Comment by cyrichardson** [comment]
Closing in favor of https://jira.rax.io/browse/INFODEV-424

**Pull request closed without merging**